core/src/main/java/cz/xtf/core/bm/BuildManager.java

```diff
@@ -16,6 +17,20 @@ public BuildManager(OpenShift openShift) {
         if (openShift.getProject(openShift.getNamespace()) == null) {
             openShift.createProjectRequest();
             openShift.waiters().isProjectReady().waitFor();
+
+            OpenShift adminOcp = OpenShifts.admin();
```

Comment:
when I use `xpaasqe` user in place of admin, it will be not null, right?

Reply:
Yeah